rclrs/Cargo.toml

```diff
@@ -30,6 +33,15 @@ tempfile = "3.3.0"
 [build-dependencies]
 # Needed for FFI
 bindgen = "0.66.1"
+# Needed for uploading documentation to docs.rs
+cfg-if = "1.0.0"
 
 [features]
+default = []
 dyn_msg = ["ament_rs", "libloading"]
+# This feature is solely for the purpose of being able to generate documetation without a ROS installation
+# The only intended usage of this feature is for docs.rs builders to work.
+generate_docs = ["rosidl_runtime_rs/generate_docs"]
```

Comment:
Should we document somewhere about how this feature works?

Reply:
I'm not sure, it depends on how much information needs to be conveyed. If it can be summarized in a couple of sentences, you might be able to get away with doing it in the comments here. Otherwise, we may need to create a separate documentation file...

Reply:
This feature's only purpose is so that docs.rs can run ros2-rust, so it's not intended to be used by users.

Reply:
But nevertheless, I agree with documenting it somewhere, I'll add something more explanatory here and we can move it elsewhere if it needs more details.

Reply:
@jhdcs can you approve this PR again if it looks good to you? I pushed a change and your review was unfortunately discarded. Thanks.